test_runner/regress/test_proxy.py

```diff
@@ -105,3 +107,12 @@ def test_forward_params_to_client(static_proxy: NeonProxy):
             for name, value in cur.fetchall():
                 # Check that proxy has forwarded this parameter.
                 assert conn.get_parameter_status(name) == value
+
+
+def test_close_on_connections_exit(static_proxy: NeonProxy):
+    with static_proxy.connect(options="project=irrelevant"):
+        with static_proxy.connect(options="project=irrelevant"):
+            if static_proxy._popen:
```

Comment:
you can just use 

```python
assert static_proxy._popen
```

It will make `mypy` happy and add one more check in order to pass the test.

With this `if` statement, you will test nothing and `pytest` will mark this test as `passed`. IMO, it's a wrong behavior :)

Reply:
Good point, changed it